dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -983,6 +983,47 @@
         #     variable: true
         #     variableDocument: true
 
+      # Camunda Exporter ----------
+      # An example configuration for the camunda exporter:
+      #
+      # These setting can also be overridden using the environment variables "ZEEBE_BROKER_EXPORTERS_CAMUNDAEXPORTER_..."
+      #
+      # camundaExporter:
+        # className: io.camunda.exporter.CamundaExporter
+        #
+        # args:
+        #   connect:
+        #     type: elasticsearch
+        #     url: http://localhost:9200
+        #     clusterName: elasticsearch
+        #     dateFormat: yyyy-MM-dd'T'HH:mm:ss.SSSZZ
+        #     socketTimeout: 1000
+        #     connectTimeout: 1000
```

Comment:
❓ What is the difference here?

What is the difference to the request timeout we had before? https://github.com/camunda/camunda/pull/23754/files#diff-6783f1e71cace45bffaa5446fc6bde1f63deb6ae0ff99fd28eeae7e659881253L845

Can we also specify the unit in the name? Otherwise it is not fully clear.

Like socketTimeoutMs